docs/using/index.rst

```diff
@@ -21,6 +21,7 @@ This Getting Started guide will walk you step-by-step through installing Flocker
    security
    config/index
    administering/index
-   tutorial/index
+   tutorial/movingvolume
+   tutorial/index  
```

Comment:
Having both `tutorial/index` and `tutorial/movingvolume` which isn't a child of the former seems strange. But if the former is going away eventually, it probably isn't worth worrying about.